libcaf_net/caf/net/stream_transport.hpp

```diff
@@ -252,7 +313,7 @@ class stream_transport {
   size_t written_;
 
   endpoint_manager* manager_;
-};
+}; // namespace net
```

Comment:
How did that comment slip in here?

```suggestion
};
```